lib/redmine_auto_assign_group/user_patch.rb

```diff
@@ -1,6 +1,6 @@
 module RedmineAutoAssignGroup
   module UserPatch
-    def save(*args, &block)
+    def save(**options, &block)
```

Comment:
It looks good on Rails 6.1, but not on Rails 5.2 for Redmine 4.2.
https://github.com/rails/rails/blob/5-2-stable/activerecord/lib/active_record/persistence.rb#L307

It should branch according to the version of Redmine.

Reply:
I am just a bash-scripting sysadmin. Searched a while, but I could only implement this by duplicating the whole function :-(
This is not C, and there's to preprocessor, so a macro won't help either I guess.

I am sure Ruby has more elegant ways to do this.
Sorry I am of no help here!

Reply:
Ok. I'll add a condition for Redmine 4.2.
Thank you for your help!

Reply:
I check this PR on Redmine 4.2.10.stable, and not found plobrems.